helm/hpcc/templates/dali.yaml

```diff
@@ -27,17 +28,34 @@ spec:
                 {{ include "hpcc.configArg" . }}
               ]
 {{ include "hpcc.addSentinelProbes" . | indent 8 }}
-{{- include "hpcc.addSecurityContext" (dict "root" $ "me" .) | indent 8 }}
+{{- include "hpcc.addSecurityContext" $commonCtx | indent 8 }}
 {{- include "hpcc.addResources" (dict "me" .resources) | indent 8 }}
-{{ include "hpcc.addImageAttrs" (dict "root" $ "me" .) | indent 8 }}
+{{ include "hpcc.addImageAttrs" $commonCtx | indent 8 }}
         volumeMounts:
 {{ include "hpcc.addConfigMapVolumeMount" . | indent 8 }}
-{{ include "hpcc.addDaliVolumeMount" $ | indent 8 }}
+{{ include "hpcc.addDaliVolumeMount" $commonCtx | indent 8 }}
 {{ include "hpcc.addSecretVolumeMounts" (dict "root" $ "categories" (list "system" ) ) | indent 8 }}
+{{- range .services -}}
```

Comment:
There should be an entry in the config for dali (and all sasha config files) that contains the vault information.  Mentioned in Shamser's PR as well.

Reply:
there are no secrets/vaults generated for Dali in master at the moment.
I don't think Dali has needed any, until now with introduction of sasha services that may.

I can wait though for Shamser's PR to be merged, so it doesn't clash etc.., then use a common list.